examples/guard_functions/src/guards.ts

```diff
@@ -54,38 +35,19 @@ class CustomError extends Error {
     }
 }
 
-export function throwCustomError(): GuardResult {
+export function throwCustomError() {
     console.log('throwCustomError called');
     throw new CustomError(
         'Execution halted by "throw custom error" guard function'
     );
 }
 
-export function preventUpgrades(): GuardResult {
+export function preventUpgrades() {
     console.log('preventUpgrades called');
-    return { Err: 'Upgrades to this canister are disabled' };
-}
-
-export function returnInvalidType(): GuardResult {
-    console.log('returnInvalidType called');
-    // @ts-ignore
-    return 'Something other than a guard result';
-}
-
-export function returnNonGuardResultObject(): GuardResult {
-    console.log('returnNonGuardResultObject called');
-    // @ts-ignore
-    return { badProp: 'Something other than a guard result' };
-}
-
-export function returnNonNullOkValue(): GuardResult {
-    console.log('nonNullOkValue called');
-    // @ts-ignore
-    return { Ok: 'Something other than null' };
+    return 'Upgrades to this canister are disabled';
```

Comment:
Shouldn't this `throw`?

Reply:
Not needed I'll just delete it in my next PR